lib/paper_trail/config.rb

```diff
@@ -5,30 +5,12 @@ module PaperTrail
   class Config
     include Singleton
     attr_accessor :timestamp_field, :serializer, :version_limit
-    attr_reader :serialized_attributes
+    attr_accessor :serialized_attributes # Kept around for backwardscompatibility
```

Comment:
The mutator method, `#serialized_attributes=`, should also have a deprecation warning.